### PR TITLE
🏃Add controller version in logs

### LINKED ISSUE #2801: Print controller versions on startup
/kind feature

**Describe the solution you'd like**
Would be great to to see the controller version in the logs, makes debugging issues for end users much simpler.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api: 0.3.*

### PR DESCRIPTION
**Pull request by tahsinrahman**
**What this PR does / why we need it**:
Print controller version in the logs 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2801

### CODE REVIEW

File: bootstrap/kubeadm/main.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ import (
 	kubeadmbootstrapv1alpha2 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha2"
 	kubeadmbootstrapv1alpha3 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3"
 	kubeadmbootstrapcontrollers "sigs.k8s.io/cluster-api/bootstrap/kubeadm/controllers"
+	"sigs.k8s.io/cluster-api/cmd/version"
```
Comment (detiber) [inline bootstrap/kubeadm/main.go]: It "feels" a bit weird to import `cmd/version` here, I'm wondering if the implementation should be moved somewhere else in the tree?

Comment (tahsinrahman) [reply bootstrap/kubeadm/main.go]: @detiber what do you suggest? move to `util`?

Comment (vincepri) [reply bootstrap/kubeadm/main.go]: Maybe open an issue to tackle this during the next iteration, we shouldn't move exported packages in a v0.3.x release

I'm fine using this package for now

Comment (tahsinrahman) [reply bootstrap/kubeadm/main.go]: created issue here https://github.com/kubernetes-sigs/cluster-api/issues/2823

**Comment by JoelSpeed** [comment]
The CI logs look like the ldflags are being injected correctly, I'm happy with this, thanks @tahsinrahman 

/lgtm

**Pull request merged**